ui/webui/test/prepare-updates-img

```diff
@@ -1,21 +1,28 @@
 #!/bin/sh
 
-set -e
+set -eu
 
 # FIXME boot.iso on rawhide does not currently contain the new cockpit dependencies
 # This will change once we include this changes upstream and start building boot.iso with the new dependencies
 # Then we can safely remove this workaround
-(
-    # store these separately so we don't need to download them each time
-    mkdir -p tmp/extra-rpms
-    cd tmp/extra-rpms
-    test -e tmp/rpms/cockpit-ws*.rpm || curl -LO https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-ws-259-1.fc35.x86_64.rpm
-    test -e cockpit-bridge*.rpm || curl -LO https://kojipkgs.fedoraproject.org//packages/cockpit/259/1.fc35/x86_64/cockpit-bridge-259-1.fc35.x86_64.rpm
-)
+COCKPIT_URLS='
+    https://kojipkgs.fedoraproject.org//packages/cockpit/275/1.fc38/x86_64/cockpit-ws-275-1.fc38.x86_64.rpm
+    https://kojipkgs.fedoraproject.org//packages/cockpit/275/1.fc38/x86_64/cockpit-bridge-275-1.fc38.x86_64.rpm
+'
```

Comment:
I agree with Martin that we should just take latest here. We are no longer trying to build the very foundations, so lagging behind cockpit no longer buys us time, instead it buys us debt... That said, we can take that step once this unclogs the queue :-)

Reply:
Agreed - we should keep this in mind but can handle it later. :)

Reply:
It's not about not moving forward, but rather about not having "moving parts" that can change between test runs.  Part of cockpit CI design is that the test environment doesn't change in any way between runs.

I had a thought about what we might be able to do here, and one of the best ways forward would be to somehow download the cockpit dependencies at the time we download the .iso and bundle them together somehow.  That would mean that we get regular cockpit updates, as part of the normal image refreshes, but would never get a shifting environment without taking explicit action.

Reply:
Ah, that's a good idea.

It's a shame that one can't use dependabot on rpms, there are many places that would be helpful.